win_images/win_packaging.ps1

```diff
@@ -14,14 +14,16 @@ retryInstall git archiver psexec golang mingw StrawberryPerl; Check-Exit
 # Update service is required for dotnet
 Set-Service -Name wuauserv -StartupType "Manual"; Check-Exit
 
-# dotnet is required for wixtoolset
-# Allowing chocolaty to install dotnet breaks in an entirely
-# non-debuggable way.  Workaround this by installing it as
-# a server-feature first.
-Install-WindowsFeature -Name Net-Framework-Core; Check-Exit
-
-# Install wixtoolset for installer build & test.
-retryInstall wixtoolset; Check-Exit
+# Install dotnet as that's the best way to install WiX 4+
+# Choco does not support installing anythong over WiX 3.14
```

Comment:
If you need to resubmit (for the `heat` thing), might as well `s/anythong/anything/`